nucliadb_core/src/lib.rs

```diff
@@ -72,6 +72,15 @@ impl From<i32> for Channel {
     }
 }
 
+impl From<Channel> for i32 {
+    fn from(value: Channel) -> Self {
+        match value {
+            Channel::STABLE => 0,
+            Channel::EXPERIMENTAL => 1,
+        }
+    }
+}
+
```

Comment:
If this is to keep compatibility with the channel defined in the protos maybe we should do it in a safer way..
An approach I am thinking is to implement `From<ReleaseChannel>` for this `Channel` and then implement `From<Channel> for i32` using the `ProtosChannel` as intermediary. What do you think? 
Is just because I find concerning having a hardcoded mapping that depends in some library implementation.

Reply:
Changed